js/tilesSuperhero.js

```diff
@@ -0,0 +1,227 @@
+const NullTile = {
+ +  canEnterFromTheTop: function(player) {return false;},
+ +  canEnterFromTheBottom: function(player) {return false;},
+ +  canEnterFromTheRight: function(player) {return false;},
+ +  canEnterFromTheLeft: function(player) {return false;},
+ +  playerLeaves: function(player){},
+ +  placeAtIn: function(){return this;},
+ +  showIn: function() {},
+ +  visit: function() {},
+ +  viewFromTheTop: function() {},
```

Comment:
I will not allow this duplicated code. Please add your tile to the tiles.js file.